phalcon/validation.zep

```diff
@@ -247,25 +247,10 @@ class Validation extends Injectable implements ValidationInterface
 
 		for validator in validators {
 			if validator instanceof ValidatorInterface {
-				if typeof field == "array" {
-					// Uniqueness validator for combination of fields is handled differently
-					if validator instanceof CombinedFieldsValidator {
-						let this->_combinedFieldsValidators[] = [field, validator];
-					}
-					else {
-						for singleField in field {
```

Comment:
@sergeyklay remove from this method `var singleField` when you will be doing some commit.